lib/sort.js

```diff
@@ -36,6 +38,15 @@ module.exports = function sortDocs(comments, options) {
   var fixed = options.toc.filter(function (val) {
     return typeof val === 'object' && val.name;
   }).map(function (val) {
+    if (typeof val.file === 'string') {
+      var filename = path.join(process.cwd(), val.file);
```

Comment:
I've changed it so that
- if loading through the config, the file is resolved by that path. 
- sort.js checks for absolute paths and uses them if found, otherwise it resolves against `process.cwd()`